torch/csrc/jit/script/compiler.cpp

```diff
@@ -1699,23 +1741,41 @@ std::shared_ptr<SugaredValue> SimpleValue::attr(SourceRange loc, Method & m, con
       Symbol::aten(field), NamedValue(loc, "self", value));
 }
 
-std::vector<Value*> inlineCallTo(Graph& g, Graph& callee, ArrayRef<Value*> inputs) {
+std::vector<Value*> inlineCallTo(Graph& g, Graph& originalCallee, ArrayRef<Value*> inputs) {
+  auto callee = originalCallee.copy();
   std::unordered_map<Value*, Value*> value_map;
   auto value_map_func = [&](Value* v) { return value_map.at(v); };
-  JIT_ASSERT(callee.inputs().size() == inputs.size());
+  JIT_ASSERT(callee->inputs().size() == inputs.size());
   for (size_t i = 0; i < inputs.size(); ++i) {
-    value_map[callee.inputs()[i]] = inputs[i];
+    value_map[callee->inputs()[i]] = inputs[i];
   }
-  for (auto* node : callee.nodes()) {
+
+  // Inlined graphs need to expose their EntryWorld and ExitWorld nodes, so that
+  // the caller graph's world token can be threaded through the callee graph.
+  //
+  // The actual joining of the world token "threads" will be done in the
+  // AnnotateEffects pass.
+  //
+  // TODO(suo): a cleaner way to handle this might be to have inlined graphs
+  // be a special prim::Inlined node, so that we can clone the entire block's
+  // metadata and don't need to expose EntryWorld and ExitWorld like this.
+  auto* storeWorld = g.insertNode(g.create(prim::StoreWorld));
```

Comment:
The words Store and Load and store are flipped here.  A load should have an output. A store should have an input.

Reply:
I'm a little confused by the terms: right now, graphs start with a `prim::Store` (converted from a `prim::Param`) and end with a `prim::Load` (converted from `prim::Return`), so I was just following that convention. Should I change that too? (I agree that your suggested meaning makes more sense to me)

Reply:
Yeah, those instructions are flipped as well.